### ISSUE TITLE
Fish completion and PKGEXT

### ISSUE THREAD

**Issue by sboukortt** [actionless/pikaur#49]
Hi, I just noticed that the fish completion for pikaur does not consider filenames ending in `.pkg.tar` when completing `pikaur -U`. Looking at `/usr/share/fish/completions/pikaur.fish`, I noticed this segment:

~~~~sh
# Upgrade options
# Theoretically, pacman reads packages in all formats that libarchive supports
# In practice, it's going to be tar.xz or tar.gz
# Using "pkg.tar.*" here would change __fish_complete_suffix's descriptions to "unknown"
complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.xz)' -d 'Package file'
complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.gz)' -d 'Package file'
~~~~

I understand not wanting to list every single possibility (though I just tried `"pkg.tar*"` and did not notice the difference mentioned in the comment), but `.pkg.tar` seems to be quite a popular option, since it bypasses compression entirely, which makes it very fast.

See for example:

- https://bbs.archlinux.org/viewtopic.php?id=127894
- https://leedev.org/arch-linux-disable-makepkg-compression/
-
 https://www.reddit.com/r/archlinux/comments/3ctiou/lpt_speed_up_makepkg_by_skippingparallelizing/
- https://wiki.archlinux.org/index.php/Makepkg#Use_other_compression_algorithms
- https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=unity-editor&id=1cab5b172f4459ea5b912a093d52a72d4bdca1e5#n50 (line 50)

Could we perhaps add it to the fish completion?

Thanks.

**Comment by actionless** [comment]
yup, feel free to submit a PR with such change

**Comment by actionless** [comment]
closed in #50
